nx-X11/programs/Xserver/hw/nxagent/NXdixfonts.c

```diff
@@ -999,1393 +622,86 @@ doListFontsAndAliases(ClientPtr client, LFclosurePtr c)
     return TRUE;
 }
 
+
 int
-ListFonts(ClientPtr client, unsigned char *pattern, unsigned length, 
```

Comment:
ListFonts() in NXdixfonts.c:

> Modified, but not kept.

Only change a see is a fprintf() encapsulated in NXAGENT_DEBUG.